test/upgrade/upgrade_test.go

```diff
@@ -51,6 +51,7 @@ func TestEventingUpgrades(t *testing.T) {
 			},
 			UpgradeWith: []pkgupgrade.Operation{
 				installation.GitHead(),
+				installation.PostInstall(),
```

Comment:
Will downgrade be a problem?

Reply:
I don't think so, but for a complete downgrade, we should run post-install jobs even after downgrading.

Reply:
Done!